resources/assets/img/icons/clear.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" width="24" height="24"><style>tspan{white-space:pre}.shp0{fill:none}.shp1{fill:#fff}</style><g id="icons/clear/white"><path id="clear-button-bg" class="shp0" d="M0 0h24v24H0V0z"/><path id="Shape" class="shp1" d="M19 6.4L17.6 5 12 10.6 6.4 5 5 6.4l5.6 5.6L5 17.6 6.4 19l5.6-5.6 5.6 5.6 1.4-1.4-5.6-5.6L19 6.4z"/></g></svg>
```

Comment:
I assume that when we merge https://github.com/elifesciences/pattern-library/pull/958 prior to this, we'll re run `bin/update` or similar to get the absolutely most recent `master` of `pattern-library` in.